File: Cpp Programs/q01.cpp
#include <iostream>
using namespace std;

int main() {
    // int num1, num2, num3;
    int num[3];
    for(int i = 0; i < 3; i++){
        cout << "Enter the number " << i << " :  ";
        cin >> num[i];
    }
    int larger = (num[0] >= num[1] && num[0] >= num[2]) ? num[0] : (num[1] >= num[0] && num[1] >= num[2]) ? num[1] : num[2];
    cout << "Largest number is  "<< larger;
    return 0;
}

Enter the number 0 :  4
Enter the number 1 :  5
Enter the number 2 :  9
Largest number is  9

File: Python Programs/q01.py
num1 = int(input("Enter a int number 1 :  "))
num2 = int(input("Enter a int number 2 :  "))
num3 = float(input("Enter a floating number 1 :  "))
num4 = float(input("Enter a floating number 2 :  "))

def arth_opr(num1, num2):
    print(f"Arthimetic Operations on {num1} and {num2} :-")
    print("Sum is : ",num1 + num2)
    print("Difference is : ", num1 - num2)
    print("Product is : ", num1 * num2)
    print("Division is : ",num1 / num2)
    print("Modulus is : ",num1 % num2)
    print("Exponent is : ", num1 ** num2)
    print("Floor division is : ", num1 // num2)

arth_opr(num1, num2)
arth_opr(num3, num4)
